examples/test-cra/package.json

```diff
@@ -17,19 +17,19 @@
     "react-dom": "^15.5.4"
   },
   "devDependencies": {
-    "@storybook/addon-actions": "file:../../addons/actions",
-    "@storybook/addon-centered": "file:../../addons/centered",
-    "@storybook/addon-info": "file:../../addons/info",
-    "@storybook/addon-knobs": "file:../../addons/knobs",
-    "@storybook/addon-links": "file:../../addons/links",
-    "@storybook/addon-notes": "file:../../addons/notes",
-    "@storybook/addon-options": "file:../../addons/options",
-    "@storybook/addon-storyshots": "file:../../addons/storyshots",
-    "@storybook/addons": "file:../../lib/addons",
-    "@storybook/channel-postmessage": "file:../../lib/channel-postmessage",
-    "@storybook/channels": "file:../../lib/channels",
-    "@storybook/react": "file:../../app/react",
-    "@storybook/ui": "file:../../lib/ui",
+    "@storybook/addon-actions": "^3.0.1",
```

Comment:
Previously there was an advantage to `test-cra` because it flattened dependencies (with npm<5) and thus "looked like" a real app in terms of how `require/import` resolved.

However, in `npm@5`, `file://` URLs are treated like `npm link`s (AFAICT), so that advantage is gone. It seems like we can just about get away with it with aggressive use of `peerDependencies` in packages and linking *every* storybook related package in the app (this is OK).

So I don't think there's any point in distinguishing `test-cra` from `cra-storybook` any more.